samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/servlet/delivery/LoginServlet.java

```diff
@@ -135,7 +135,10 @@ public void doPost(HttpServletRequest req, HttpServletResponse res)
     }
     delivery.setAssessmentId(pub.getPublishedAssessmentId().toString());
     delivery.setAssessmentTitle(pub.getTitle());
-    delivery.setHonorPledge(pub.getAssessmentAccessControl().getHonorPledge());
+    Boolean honorPledge = pub.getAssessmentAccessControl().getHonorPledge();
+    if (honorPledge == null)
+	honorPledge = false;
```

Comment:
boolean honorPledge = BooleanUtils.toBooleanDefaultIfNull(pub.getAssessmentAccessControl().getHonorPledge(), false);

would be cleaner. (And who doesn't love Apache Commons?) :)